.ci/packer_cache.sh

```diff
@@ -28,26 +31,24 @@ haproxy:1.9
 jruby:9.1
 jruby:9.2
 maven:3.5.3-jdk-10
-mcr.microsoft.com/dotnet/core/aspnet:2.2
+mcr.microsoft.com/dotnet/core/aspnet:2.2-alpine
 mcr.microsoft.com/dotnet/core/sdk:2.2
 node:10
 node:10.0
-node:11
-node:11.0
 node:12
 node:12.0
 node:13
 node:13.0
 node:14
 node:14.0
-node:6
-node:6.0
-node:8
 node:8-slim
-node:8.1
```

Comment:
if you remove this one here probably we have to add it to the Node.js repo

Reply:
`8.1` is not an active version anymore:
- https://github.com/elastic/apm-agent-nodejs/blob/master/.ci/.jenkins_nodejs.yml

Caused by:
- https://github.com/elastic/apm-agent-nodejs/commit/408bffdeb4e44ad26f5f4086720ba725ebf19db8#diff-fe5284f01f6313792b5e5defc5b319c2